src/commands.ts

```diff
@@ -42,12 +44,51 @@ export default function registerCommands(commandObserver: CommandObserver, packa
 					});
 				});
 		}),
+		vscode.commands.registerCommand('pgproj.deploy.current', async (args) => {
+			requireDotNetSdk(dotNetSdkVersion).then(
+				async dotnet => {
+					await deployCurrentProject(args, dotnet, commandObserver);
+				});
+		}),
 		vscode.commands.registerCommand('pgproj.add.new', async (args) => {
-			await addNewPostgreSQLProject(args, packageInfo.maxSupportedPostgreSQLProjectSDK, commandObserver);
+			await addNewPostgreSQLProject(args, packageInfo.maxSupportedPostgreSQLProjectSDK);
 		})
 	];
 }
 
+async function deployCurrentProject(args, dotNetSdk: DotNetInfo, commandObserver:  CommandObserver, cancelToken?: vscode.CancellationToken) {
+	let project = '';
+	if (!args) {
+		const activeEditor = vscode.window.activeTextEditor;
+		if (activeEditor !== undefined) {
+			project = activeEditor.document.uri.fsPath;
+		}
+	} else {
+		project = args.fsPath;
+	}
+
+	await buildProjects(dotNetSdk, [project], commandObserver, cancelToken).then(async result => {
+		if (result && !result.some(b => b.status === buildStatus.Failure || b.status === buildStatus.Skipped)) {
+			var projectName = path.basename(project, '.pgproj');
```

Comment:
We may want output name to be configurable eventually. I think the common approach is to run msbuild with specific target that returns the name of the output file, something like ```dotnet build -t GetOutputFilePath``` and use that.
If I understand correctly - we don't allow user to change the name yet, so we can do it properly later.

Reply:
Ah okay, I didn't know we can do that. Lets keep it this way for now and I will change it in future, meanwhile I will figure out how to use this -t flag